packages/desktop-client/src/components/LoggedInUser.js

```diff
@@ -34,80 +38,98 @@ function LoggedInUser({
     window.__history.push('/change-password');
   }
 
-  function onMenuSelect(type) {
+  async function onMenuSelect(type) {
     setMenuOpen(false);
 
     switch (type) {
       case 'change-password':
         onChangePassword();
         break;
+      case 'sign-in':
+        await closeBudget();
+        window.__history.push('/login');
+        break;
       case 'sign-out':
         signOut();
         break;
+      case 'config-server':
+        await closeBudget();
+        window.__history.push('/config-server');
+        break;
       default:
     }
   }
 
-  async function onClick() {
+  function serverMessage() {
     if (!serverUrl) {
-      await closeBudget();
-      window.__history.push('/config-server');
-    } else if (userData) {
-      setMenuOpen(true);
-    } else {
-      await closeBudget();
-      window.__history.push('/login');
+      return 'No server';
     }
+
+    if (userData?.offline) {
+      return 'Server offline';
+    }
+
+    return 'Server';
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
    return 'Connected to server';
```

Reply:
I wanted to keep it short since it will show up a lot, but I am open to lengthening it. What do you think?

Reply:
Maybe just "Connected"?

Reply:
Then it’s not clear that it’s about the server. I think since the typical case is “server connected” (especially since we don’t have service workers yet) it should be shortened to just say “server” in this case since it acts as a menu.

Reply:
If I'm looking at the UI and I see "server" - I would not understand what it means.

So maybe lets make it "server online" or "server connected"? We already have "Server offline"

Reply:
Ok, server online is fine with me